eoan/arches

```diff
@@ -0,0 +1,5 @@
+amd64
+arm64
+armhf
+ppc64el
```

Comment:
So should we add `i386` back here?

Reply:
We still seem to be producing i386 images for eoan so yes, let's put it back for now.